content/generic.rst

```diff
@@ -21,6 +21,9 @@ something is wrong.
 
 * Rule #3: If it ain't broke, don't fix it!
 
+* Ammendment to Rule #3: If it's not totally broken but still wastes a lot of your time, fix it! 
```

Comment:
We also have each month a happy hacking day, in which all ticket priorities are ignored and you can work on anything that you want, including fixing low priority bugs or improving our tools

the happy hacking day is documented in our internal wiki, but I will look at moving it to this public place